src/containers/Login.js

```diff
@@ -11,7 +10,7 @@ export default class Login extends Component {
         <TextInput style={styles.input} placeholder="Username" />
         <TextInput style={styles.input} placeholder="Password" />
         <TouchableOpacity
-          onPress={() => { console.log('Login!!!!') }}
+          onPress={() => navigate("Tab")}
```

Comment:
Here instead of `navigate("Tab")` , we need to dispatch an action that will check the credentials of the form, then rendering the Profile/Dashboard tab navigator.